app/models/portfolio.rb

```diff
@@ -14,6 +14,10 @@ class Portfolio < ApplicationRecord
   validates_presence_of :name
   validates_presence_of :description
 
-  has_many :portfolio_items
+  has_and_belongs_to_many :portfolio_items
 
+  def add_portfolio_item(portfolio_item_id)
+    portfolio_item = PortfolioItem.find_by(id: portfolio_item_id)
+    portfolio_items << portfolio_item
+  end
```

Comment:
I'm not sure I like this method as it feels like it's wrapping basic built-in Rails stuff.

Reply:
@Fryguy - That method will be morphing quite a bit once we handle catalog_items. So basically just a place holder for the api right now.

Reply:
@Fryguy - Plus I needed something to actually test that wasn't basic rails. 😆

Reply:
haha, that's fine, then